src/Bicep.Core.UnitTests/Assertions/BaselineHelper.cs

```diff
@@ -18,7 +18,7 @@ public static class BaselineHelper
 
         private const string SetBaseLineSettingName = "SetBaseLine";
 
-        public static bool ShouldSetBaseline(TestContext testContext) =>
+        public static bool ShouldSetBaseline(TestContext testContext) => 
```

Comment:
[nit] Trailing whitespace.

Reply:
Heh. I've been adding `true || ` to make the tests run in baseline update mode in VS. Will remove in next commit.

---
In reply to: [582214047](https://github.com/Azure/bicep/pull/1521#discussion_r582214047) [](ancestors = 582214047)